CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ option(BUILD_LMDB "Build LMDB readers" OFF)
 option(BUILD_TENSORFLOW "Build TensorFlow plugin" OFF)
 option(BUILD_JPEG_TURBO "Build with libjpeg-turbo" ON)
 option(BUILD_NVJPEG "Build with nvJPEG" ON)
+option(BUILD_NIGHTLY "Build nightly version of DALI" OFF)
```

Comment:
This is nitpicking, but BUILD_SOMETHING for me indicates that we turn something on/off in the project, exclude some functionality, etc, and not just change the suffix of the build name. As I look at it, this is probably just a matter of naming conventions and documentation.

I am not sure if we really need to make the CMake know, it is building a nightly build, and maybe just pass a -DVERSION_SUFFIX or -DBUILD_SUFFIX from {weekly, nightly, rc-0, rc-1, ...}?
Not exactly sure how to tie it with dali/VERSION file.

Reply:
As long as CMake is responsible for preparing the manifest for whl that it is the best place for embedding that logic there.